1/dn01/src/dn01.jl

```diff
@@ -0,0 +1,169 @@
+module dn01
+
+export RedkaMatrika, getindex, setindex!, firstindex, lastindex, sor
+
+"""
+    RedkaMatrika je predstavljena z matrikami V, ki hrani neničelne elemente po vrsticah
```

Comment:
```suggestion
    Podatkovni tip `RedkaMatrika` je predstavljen z matriko V, ki hrani neničelne elemente po vrsticah
```

Reply:
Bolje kot opis implementacije, ki je tako ali tako očitna iz kode, je opisati namen definicije novega podatkovnega tipa:
```suggestion
    RedkaMatrika je podatkovni tip, ki omogoča definicijo matrik z majhnim številom neničelnih elementov na prostorsko učinkovit način. Njegova uporaba omogoča znaten prihranek pomnilnika in hitrejše izvajanje osnovnih računskih operacij in algoritmov na redkih matrikah. 
```